docs/jwst_large_psf.ipynb

```diff
@@ -220,18 +220,54 @@
    "id": "5344b042-586b-4d2b-930c-8e54d945e94d",
    "metadata": {},
    "source": [
-    "In this example we are going to simulate the same PSF as before but using a 4x larger JWST pupil. As part of the required data files for WebbPSF ([webbpsf-data](https://stsci.box.com/shared/static/qxpiaxsjwo15ml6m4pkhtk36c9jgj70k.gz)), the user has access to four different sizes:\n",
+    "In this example we are going to simulate the same PSF as before but using a 4x larger JWST pupil in terms of the number of pixels. As part of the required data files for WebbPSF ([webbpsf-data](https://stsci.box.com/shared/static/qxpiaxsjwo15ml6m4pkhtk36c9jgj70k.gz)), the user has access to four different sizes:\n",
     "- jwst_pupil_RevW_npix1024.fits.gz (default)\n",
     "- jwst_pupil_RevW_npix2048.fits.gz\n",
     "- jwst_pupil_RevW_npix4096.fits.gz\n",
     "- jwst_pupil_RevW_npix16384.fits.gz\n",
     "\n",
-    "In this case, npix refers to the number of pixels scaled from the same pupil side length (~ 6.6 meters). "
+    "In each case, npix refers to the number of pixels used to sample across the same pupil side length (~ 6.6 meters). \n",
+    "\n",
+    "To perform a large PSF simulation we need to provide the name of one of the larger size JWST pupil files via the `pupil` attribute of the instrument. \n",
+    "\n",
+    "Then, as before, we use the function `load_wss_opd_by_date` or, if you know the appropriate OPD, `load_wss_opd`. These functions will take the size of the user-specified JWST pupil and scaled the observed wavefront measurement (OPD) accordingly."
+   ]
```

Comment:
@mperrin There is a new cell in the notebook (cell 5) that seems more like a "test" cell , in order to check the pupil parameters, but not necessarily intended for the documentation, is this correct? Unless we add some text, e.g.,  " Before changing the pupil file we can check the default values" , or something like that. 

`nrc = webbpsf.NIRCam()
print(nrc.pupil)

webbpsf.optics._get_initial_pupil_sampling(nrc), nrc._datapath`

Reply:
I had to debug and make a fix in a file elsewhere in webbpsf to get the notebook to run correctly. In my tests, it was erroneously looking in the wrong directory for pupil files specified with a relative path. I'm not sure how this worked for you before, since it didn't work for me... In any case that is test code and can come out

Reply:
I removed that extra cell, and pushed the revised notebook